### ISSUE TITLE
clarification on treatmentCitation

### ISSUE THREAD

**Issue by punkish** [punkish/zenodeo#15]
given this XML

```xml
<subSubSection type="reference_group">
<paragraph>
<treatmentCitationGroup>
<taxonomicName authorityName="Meyrick" authorityYear="1915" class="Insecta" family="Gracillariidae" genus="Phyllocnistis" higherTaxonomySource="GBIF" kingdom="Animalia" order="Lepidoptera" pageId="22" pageNumber="323" phylum="Arthropoda" rank="species" species="sciophanta">
<emphasis>Phyllocnistis sciophanta</emphasis>
</taxonomicName>
; 
<treatmentCitation author="Meyrick" year="1915">
<bibRefCitation author="Meyrick" refString="Meyrick, E. (1915 b) Descriptions of South American Micro-Lepidoptera. Transactions of the Entomological Society of London, 1915 (2), 201 - 256. https: // doi. org / 10.1111 / j. 1365 - 2311.1915. tb 02527. x" type="journal article" year="1915" yearSuffix="b">Meyrick 1915b</bibRefCitation>
: 241
</treatmentCitation>
. 
<treatmentCitation author="Davis" year="1984">
<bibRefCitation author="Davis" refString="Davis, D. R. &amp; Miller, S. E. (1984) Gracillariidae. In: Heppner, J. B. (Ed.), Atlas of Neotropical Lepidoptera, Checklist. Part 1. Dr. W. Junk Publishers, The Hague, pp. 25 - 27." type="book chapter" year="1984">Davis &amp; Miller 1984</bibRefCitation>
: 27
</treatmentCitation>
. De Prins 
<emphasis>et al</emphasis>
. 2016: 37.
</treatmentCitationGroup>
</paragraph>
</subSubSection>
```

What should the **treatmentCitations** be? Are the following correct?

1. Phyllocnistis sciophanta Meyrick 1915 sec. Meyrick 1915b
2. Phyllocnistis sciophanta Meyrick 1915 sec. Davis &amp; Miller 1984

@mguidoti @myrmoteras

**Comment by mguidoti** [comment]
I would say: 

_Phyllocnistis sciophanta_ Meyrick, 1915
_Phyllocnistis sciophanta_ Meyrick, 1915 sec. Davis & Miller, 1984

There is no need to say sec. Meyrick, 1915b as this is original description and thus, should remain with the taxon authority only. 

How do I know it's the original description? Because it's the first bibref citation under the treamentCitationGroup (traditionally, the first treatment citations is always the original description) and it's the same author and year of the taxon. 

I would also suggest to pay attention to commas in between author(s) names and authority years...

Let me know if this helps, Puneet. 

Best,

**Comment by punkish** [comment]
Thanks. Well, I am glad I asked. While it is possible I missed it earlier, as far as I recall, this is the first time I am hearing the logic of "no need to say sec. … as this is original description." @myrmoteras, can you please confirm this so I go ahead and implement it. I would like to avoid parsing 300K records and rebuilding the database needlessly… I've already done it too many times.

**Comment by punkish** [comment]
@mguidoti @mymoteras

**update:** Ok, now I have modified my program so I am getting the following

1. Phyllocnistis sciophanta Meyrick, 1915
2. Phyllocnistis sciophanta Meyrick, 1915 sec. Davis & Miller 1984

Note that the comma in the first treatment citation has been added by me. However, I can't inject a comma in the fragment *after* the 'sec.' part in the second citation because that is the verbatim text of the **bibRefCitation** tag in the second **treatmentCitation** tag. It is possible for me to inject a comma in the first citation because the **authorityName** and **authorityYear** are extracted from the attributes of the tag. So, if you want consistency, perhaps it is better to take out the comma completely, or live with how it is above.

@myrmoteras, I am waiting to hear from you. If I don't hear from you in a couple of days, I will assume you concur with this thread and I will close this matter.

Danke schön alles.

**Comment by myrmoteras** [comment]
I just asked Rich, a Plazi member and the person who runs zoobank, the registry of new zoological names operated under the International Commission of Zoological Nomenclature. He normally responds within 24 hours

**Comment by myrmoteras** [comment]
@mguidoti did you give any thought how you deal with a treatment citation that is being cited?

For example,  Orchis mascula Linnaeus 1752 sec Landolt 1973 used by Müller 1988, where he augments the previous treatments? 

This is sometimes relevant when several floras in a region exist and with it different interpretations of a species.

**Comment by punkish** [comment]
> I just asked Rich, a Plazi member and the person who runs zoobank, the registry of new zoological names operated under the International Commission of Zoological Nomenclature. He normally responds within 24 hours

Well, no matter what Rich says, please see what I wrote… I cannot inject a comma in the text I extract from a tag. I can only inject commas (or otherwise modify) when I create text strings using discrete extracted values such as attributes of tags. In the first example above (Meyrick, 1915), I can add the comma because both 'Meyrick' and '1915' are extracted as attributes. But in the second example, 'Davis & Miller 1984' is the text of the tag, and that is how it is written in the original paper, so that is how I have it. I can't add a comma because there is no rule I can make that will positively apply to every conceivable case.

**Comment by mguidoti** [comment]
Hi guys,

I'll split my answer here since we're talking about slightly different, but related, things.

@myrmoteras 

### On the comma or no comma thing
I dove deeper into this and found some interesting things. 

**Although the comma is expected to be present when representing the authority of the taxon**, Nico Franz [on at least one of his publications](https://www.biorxiv.org/content/biorxiv/early/2015/07/08/022145.full.pdf) use parenthesis to indicate the year of the treatment's authority, like a bibliographical citation. 

Moreover, the Code has some insight into this on [recommendation 51G](https://www.nhm.ac.uk/hosted-sites/iczn/code/includes/page.jsp?nfv=&article=51#3): "Recommendation 51G. Citation of person making new combination. If it is desired to cite both the author of a species-group nominal taxon and the person who first transferred it to another genus, the name of the person forming the new combination should follow the parentheses that enclose the name of the author of the species-group name (and the date, if cited; see Recommendation 22A.3).
Examples. _Limnatis nilotica_ (Savigny) Moquin-Tandon; _Methiolopsis geniculata_ (Stï¿½l, 1878) Rehn, 1957."

The Code makes this recommendation only for new combinations, but this could be adapted and applied for all treatment citations within our context/scope. Note that there are commas in both authors, and there isn't any 'sec.'.

I'm not sure what Pyle will say, but I think it might be just yet another opinion... right?

### On treatment citations being cited: 
In my opinion, it should not have any different representation, it should be treated as a regular treatment citation. Thus, in your example, _Orchis mascula_ Linnaeus, 1752 sec. Landolt, 1973 was used by Müller (1988) to produce another treatment of the same taxon concept, thus both should be referred as regular treatments: 

1. _Orchis mascula_ Linnaeus, 1752 sec. Landolt, 1973
2. _Orchis mascula_ Linnaeus, 1752 sec. Müller, 1988

If one wants to explicitly affirm the link between Müller's to Landolt's treatments, that should be done in the discussion of the paper: "Müller (1988) based his description on Landolt (1973)..."

That's how I always have seen these cases in my mind. The other solution would be add yet another level of authority on the taxonConceptLabel, to cover these cases, but I think this isn't even allowed in the OpenBiodiv, but I could be wrong - and more importantly, I don't think our *.XML carry this third level of information.

What do you think?

@punkish 

### I cannot inject a comma in the text...

I'm sorry Puneet, but I think you can add a comma if you need to, with a basic, simple and very easy string manipulation. The predictor is very clear: numbers. There are no names with numbers, at least I never seen one. Adding a comma one character before the first number appear in the string (so we can preserve the space after the inserted comma), and voilá, you'll have it. No?

You can even add an additional test: after the first number, the three following characters must also be numbers. 

And I'm sure you know that checking if a character is a number is a no-brainer as well. 

Right? 

But now we're waiting Pyle's opinion on this, and @myrmoteras as well after what I just said and bring to the table. As you're concerned about re-running your databases, if you allow me to say, I wouldn't do anything until we get this cleared. 

Best,

**Comment by punkish** [comment]
No, it is actually not that easy. If it was, I would not have said that it is difficult. There are a lot of complexities, and I’d rather not modify existing text it possible. There are a ton of edge cases, and we have only 300K records right now. These edge cases will only increase as the number of treatments increases. In any case, if that comma was so important, it would have been in the text. 

Anyway, let’s wait to resolve this.

**Comment by punkish** [comment]
> I just asked Rich, a Plazi member and the person who runs zoobank, the registry of new zoological names operated under the International Commission of Zoological Nomenclature. He normally responds within 24 hours

_Reply from Rich_

Hi Donat,
 
You caught me just in time!  I am about to get on a ship for nearly a month, so may not have been available.
 
I would eliminate all commas.
 
I would suggest the following standard:
 
“Aus bus Linnaeus 1758” [represents the name by itself, no particular treatment]
“Xus bus (Linnaeus 1758)” [represents the name by itself, no particular treatment but in a different genus]
“Aus bus Linnaeus 1758 sec Linnaeus 1758” [represents the original treatment of the name by Linnaeus]
“Aus bus Linnaeus 1758 sec Jones 1950” [represents the a subsequent treatment of the name by Jones]
“Xus bus (Linnaeus 1758) sec Smith 1958” [represents the a subsequent treatment of the name by Smith, in a different genus]
 
Of course this is not perfect because there may be more than one “Linnaeus 1758”, “Jones 1950”, or “Smith 1980”.  But that’s what the UUID identifiers are for.  These can just be labels.  I suppose you could add suffixes like “Jones 1950a”, “Jones 1950b”, etc., but these would need to be tied to a specific context (bibliography), as there are no standards for what “a” and “b” actually represent.
 
I hope that helps!
 
Aloha,
Rich
